### PR TITLE
Define module for managing the NetApp E-Series iSCSI target

### PR DESCRIPTION
**Pull request by lmprice**
Create a new module for managing E-Series iSCSI targets.

##### SUMMARY

Develop a new module for managing the iSCSI target settings for E-Series storage-systems.

##### ISSUE TYPE
 - New Module Pull Request
##### COMPONENT NAME

netapp_e_iscsi_target
##### ANSIBLE VERSION

```
ansible 2.6.0 (detached HEAD ce8c336d52) last updated 2018/04/18 11:23:03 (GMT -500)
  config file = None
  configured module search path = [u'/home/lorenp/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/lorenp/ansible/lib/ansible
  executable location = /home/lorenp/ansible/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]

```

##### ADDITIONAL INFORMATION

```
changed: [testhost] => {"alias": "abc", "changed": true, "iqn": "iqn.1992-08.com.netapp:2804.600a098000a096d2000000005a0e8f45", "msg": "The interface settings have been updated."}
```

### CODE REVIEW

**Review by ndswartz — APPROVED** [review]
shipit

File: lib/ansible/utils/module_docs_fragments/netapp.py [hunk 1]
```diff
@@ -121,24 +121,31 @@ class ModuleDocFragment(object):
   api_username:
```
Comment (gundalow) [inline lib/ansible/utils/module_docs_fragments/netapp.py]: I think this change needs removing from this PR (as it's been done in another PR which has since been merged)

Also you need to rebase to resolve the merge conflict https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html

**Review by gundalow — APPROVED** [review]
Module itself looks OK, please just remove the docs_fragment and ignore.txt files from this PR and then rebase your PR I think this change needs removing from this PR (as it's been done in another PR which has since been merged)

Also you need to rebase to resolve the merge conflict https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html
